hazelcast/src/main/java/com/hazelcast/nio/IOService.java

```diff
@@ -57,12 +58,6 @@
 
     TextCommandService getTextCommandService();
 
-    boolean isMemcacheEnabled();
```

Comment:
A lot of these properties have been replaced by direct HazelcastProperties access. This current layer of abstraction provides zero value; just makes the system more complex.